### PR TITLE
docs: fix confusion in unauthenticated access docs

### PR DESCRIPTION
**Pull request by marionschleifer**
### Description

Fix confusion in unauthenticated docs and link to webhook response section.

### Changelog

- [x] `CHANGELOG.md` is updated with user-facing content relevant to this PR. If no changelog is required, then add the `no-changelog-required` label.

### Affected components

- [x] Docs

### Affected pages

https://deploy-preview-5319--hasura-docs.netlify.app/graphql/manual/auth/authentication/unauthenticated-access.html#configuring-unauthenticated-public-access

### CODE REVIEW

File: docs/graphql/manual/auth/authentication/unauthenticated-access.rst [hunk 1]
```diff
@@ -31,21 +31,29 @@ A guide on setting up unauthenticated user permissions can be found :ref:`here <
...
 
-You can use the env variable ``HASURA_GRAPHQL_UNAUTHORIZED_ROLE`` or ``--unauthorized-role`` flag to set a role
+You can use the env variable ``HASURA_GRAPHQL_UNAUTHORIZED_ROLE`` or the ``--unauthorized-role`` flag to set a role
 for unauthenticated (non-logged in) users. See :ref:`server_flag_reference` for more details
 on setting this flag/env var.
 
+No auth setup
+^^^^^^^^^^^^^
+
+With no auth setup, an unauthenticated request is one that doesn't set an explicit role defined in permissions.
```
Comment (marionschleifer) [inline docs/graphql/manual/auth/authentication/unauthenticated-access.rst:55]: @sezgi @rikinsk I'm not quite sure if this is the best definition for an unauthenticated request with no auth setup but couldn't come up with anything else. You have any thoughts on this?

**Comment by sezgi** [comment]
Found a typo on this page under Introduction -- "unaunthenticated" -- can we fix in this PR?

File: docs/graphql/manual/auth/authentication/unauthenticated-access.rst (hunk 1 above)
Comment (sezgi) [reply docs/graphql/manual/auth/authentication/unauthenticated-access.rst:55]: I'm not sure how this works. If no role is defined in permissions, what would the env variable be matching? I'll defer to @rikinsk and @tirumaraiselvan on this one.

**Comment by marionschleifer** [comment]
@sezgi typo in introduction is fixed, thanks for catching it :)

File: docs/graphql/manual/auth/authentication/unauthenticated-access.rst [hunk 2]
```diff
@@ -22,7 +22,7 @@ Once you have configured authentication, by default Hasura GraphQL engine will r
 receives.
```
Comment (rikinsk) [inline docs/graphql/manual/auth/authentication/unauthenticated-access.rst:22]: `Once you have configured authentication` -> Once you have configured an admin-secret [link to securing your endpoint page]

File: docs/graphql/manual/auth/authentication/unauthenticated-access.rst (hunk 1 above)
Comment (rikinsk) [reply docs/graphql/manual/auth/authentication/unauthenticated-access.rst:55]: With no auth setup, every request without the admin-secret header is considered as an unauthenticated request. 

Nitpick, maybe moving "no auth setup" as the first section (befoore Webhooks) would make for a more logical reading order

**Pull request merged**